src/app/custom/translations.tsv

```diff
@@ -1099,3 +1099,6 @@
 "ejs_data"	"Variable containing the routine data"	"Variable contenant les données de la routine"
 "ejs_lodash"	"Variable containing the Lodash function"	"Variable contenant les fonctions de Lodash"
 "routine_args"	"Routine fields"	"Champs de la routine"
+"routine_arg"	"Field N°"	"Champ N°"
+"routine_arg_add"	"Add a new field"	"Ajouté un nouveau champ"
+"routine_arg_delete"	"Delete this field"	"Supprimé ce champ"
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
"routine_arg_add"	"Add a new field"	"Ajouter un nouveau champ"
"routine_arg_delete"	"Delete this field"	"Supprimer ce champ"
```